src/pages/Profile.js

```diff
@@ -98,40 +138,83 @@ export default function Profile(props) {
           if (nextPage.done) {
             setAllLoaded(true);
           } else {
-            processPosts(nextPage.value);
+            processPosts(nextPage.value, initLocal);
           }
         }
       };
       window.addEventListener("scroll", loadNext);
     }
-  }, [postLoader, allLoaded]);
+  }, [postLoader, allLoaded, initLocal]);
 
   //execute this effect on entry and when the feedDAC connection status is valid
   useEffect(() => {
-    //setLoading(true);
-    //setProfile(null);
     if (!isMySkyLoading && feedDAC.connector) {
+      //get feed using only remote means
+      const getInitFeed = async (usedLocal = false) => {
+        const postsLoader = await loadBlogProfile(
+          id.substring(8),
+          feedDAC,
+          client
+        );
+        const page0 = await postsLoader.next();
```

Comment:
nit: does `next()` always return the 0 index page? If not `nextPage` would be a better variable name.